README.md

```diff
@@ -194,45 +194,23 @@ sudo grep 'mycmd' /var/log/syslog
 <details>
 <summary>Useful Gstreamer Commands</summary>
 <br>
-Start Gstreamer AV1 Encoding (On Nvidia Jetson AGX Orin): 
+To start gstreamer client make sure to add the deepstream layer to the docker layers 
 
-```
-gst-launch-1.0 v4l2src device=/dev/video0 ! "video/x-raw,width=640,height=480,framerate=15/1" ! nvvidconv ! "video/x-raw(memory:NVMM),format=NV12" ! nvv4l2av1enc bitrate=200000 ! "video/x-av1" ! udpsink host=127.0.0.1 port=5000
-```
-
-Start Gstreamer AV1 Decoding (On Nvidia Jetson AGX Orin): 
-
-```
-gst-launch-1.0 udpsrc port=5000 ! "video/x-av1,width=640,height=480,framerate=15/1" ! queue ! nvv4l2decoder ! nv3dsink
```

Comment:
Why did you delete all these commands from the README @jmblixt3? Because I feel like they could be useful to keep around in case they are needed some time.

Reply:
Go look at an old revision they are now obsolete

Reply:
please keep them as a reference, you can move them to a different drop-down menu of the readme if you want, called "gstreamer pipeline examples" or something